config/sync/field.field.paragraph.os2loop_section_page_info_block.os2loop_section_page_title.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 25b9872e-7051-4320-837c-24c277bfbe02
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.os2loop_section_page_title
+    - paragraphs.paragraphs_type.os2loop_section_page_info_block
+id: paragraph.os2loop_section_page_info_block.os2loop_section_page_title
+field_name: os2loop_section_page_title
+entity_type: paragraph
+bundle: os2loop_section_page_info_block
+label: Title
+description: ''
+required: false
```

Comment:
Should the title be required? I vote “Yes!”

Reply:
Why not just let the user decide if they want to add a h2 or not above the text?
Not all content needs a header. There will often be a h1 right above it.
I vote "no!"